lighthouse-core/report/html/templates.html

```diff
@@ -359,14 +359,38 @@
   </style>
 
   <div class="lh-topbar">
-    <!-- Flat Lighthouse logo.  -->
-    <svg class="lh-topbar__logo" viewBox="0 0 192 192">
+    <!-- Lighthouse logo.  -->
+    <svg class="lh-topbar__logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+      <defs>
+        <linearGradient x1="57.456%" y1="13.086%" x2="18.259%" y2="72.322%" id="a">
+          <stop stop-color="#262626" stop-opacity=".1" offset="0%"/>
+          <stop stop-color="#262626" stop-opacity="0" offset="100%"/>
+        </linearGradient>
+        <linearGradient x1="100%" y1="50%" x2="0%" y2="50%" id="b">
+          <stop stop-color="#262626" stop-opacity=".1" offset="0%"/>
+          <stop stop-color="#262626" stop-opacity="0" offset="100%"/>
+        </linearGradient>
+        <linearGradient x1="58.764%" y1="65.756%" x2="36.939%" y2="50.14%" id="c">
+          <stop stop-color="#262626" stop-opacity=".1" offset="0%"/>
+          <stop stop-color="#262626" stop-opacity="0" offset="100%"/>
+        </linearGradient>
+        <linearGradient x1="41.635%" y1="20.358%" x2="72.863%" y2="85.424%" id="d">
+          <stop stop-color="#FFF" stop-opacity=".1" offset="0%"/>
+          <stop stop-color="#FFF" stop-opacity="0" offset="100%"/>
+        </linearGradient>
+      </defs>
       <g fill="none" fill-rule="evenodd">
-        <path d="M0 0h192v192H0z"/>
-        <path class="shape" d="M67.705 179.352l2.603-20.82 49.335-16.39 4.652 37.21A87.893 87.893 0 0 1 96 184a87.893 87.893 0 0 1-28.295-4.648zM52.44 172.48C25.894 157.328 8 128.754 8 96 8 47.399 47.399 8 96 8s88 39.399 88 88c0 32.754-17.894 61.328-44.44 76.48L130 96h6V80h-8V48L96 28 64 48v32h-8v16h6l-9.56 76.48zM113.875 96l2.882 23.05-43.318 14.433L78.125 96h35.75zM80 80V56.868l16-10 16 10V80H80z"/>
+        <path d="M0 0h24v24H0z"/>
+        <path d="M12 3l4.125 2.625v3.75H18v2.25h-1.688l1.5 9.375H6.188l1.5-9.375H6v-2.25h1.875V5.648L12 3zm2.201 9.938L9.54 14.633 9 18.028l5.625-2.062-.424-3.028zM12.005 5.67l-1.88 1.207v2.498h3.75V6.86l-1.87-1.19z" fill="#F44B21"/>
+        <path fill="#FFF" d="M14.201 12.938L9.54 14.633 9 18.028l5.625-2.062z"/>
+        <path d="M6 18c-2.042 0-3.95-.01-5.813 0l1.5-9.375h4.326L6 18z" fill="url(#a)" fill-rule="nonzero" transform="translate(6 3)"/>
```

Comment:
do any of the svg optimizers bake in transforms? Nice for authoring, but all of the path coordinates can just be translated by 6, 3 and the attribute can be dropped

Reply:
I don't know how to modify each of these coords. Is this important?

for fun, i just removed the transform, and it makes a cool ghost effect
![image](https://user-images.githubusercontent.com/4071474/66347549-721d2180-e909-11e9-9026-c84a8afcdea4.png)

Reply:
> I don't know how to modify each of these coords. Is this important?

no, nice to have but not at all critical

Reply:
(it would save 26 characters or whatever :)